StitchApp/RegressionTest/RegressionTestAppFunctions/compareChildJobs/source.js

```diff
@@ -0,0 +1,25 @@
+exports = function(commitHash, testJobs, prodJobs){
```

Comment:
Realm functions are a bit disembodied because they are executed in sometimes mysterious ways that aren't obvious when examining the source -- a good best practice moving forward would be to leave a comment in the code to describe the function -- what calls it and what it does. I'm confused what the locally scoped testTime, etc. does here, are we missing something?

Reply:
ah no just artifacts from when we were going to compare the time it took, num of failures, etc etc. ill delete